### PR TITLE
Changed to display recent photos first.

### PR DESCRIPTION
**Pull request by a2444mo10**
Changed to display recent photos first.
The screen transitions have also been changed.

### CODE REVIEW

File: NohanaImagePicker/AlbumListViewController.swift [hunk 1]
```diff
@@ -178,37 +185,12 @@ class AlbumListViewController: UITableViewController, EmptyIndicatable, Activity
...
-                fatalError("unexpected segue identifer")
-            }
-        }
-    }
-
     // MARK: - IBAction
 
-    @IBAction func didPushCancel(_ sender: AnyObject) {
-        if let nohanaImagePickerController = nohanaImagePickerController {
-            nohanaImagePickerController.delegate?.nohanaImagePickerDidCancel(nohanaImagePickerController)
```
Comment (tyokujin) [inline NohanaImagePicker/AlbumListViewController.swift:211]: Is there an alternative delegate  function?

Comment (a2444mo10) [reply NohanaImagePicker/AlbumListViewController.swift:211]: Modify it to be called by RootViewController.

File: NohanaImagePicker/NohanaImagePickerController.swift [hunk 2]
```diff
@@ -138,6 +154,10 @@ open class NohanaImagePickerController: UIViewController {
     open func dropAsset(_ asset: Asset) {
         _ = pickedAssetList.drop(asset: asset)
     }
+    
+    open override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        print(#function)
+    }
```
Comment (tyokujin) [inline NohanaImagePicker/NohanaImagePickerController.swift:160]: debug code?

Comment (a2444mo10) [reply NohanaImagePicker/NohanaImagePickerController.swift:160]: yes, debug code.
I'll fix it.

**Review by tyokujin — APPROVED** [review]

**Pull request merged**